examples/with-redux-observable/pages/index.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
 import Link from 'next/link'
 import withRedux from 'next-redux-wrapper'
-import initStore from '../lib'
-import { startFetchingCharacters, stopFetchingCharacters } from '../lib/reducer'
-import * as api from '../lib/api'
+import initStore from '../redux'
 import CharacterInfo from '../components/CharacterInfo'
+import { rootEpic } from "../redux/epics";
+import * as actions from "../redux/actions";
+import { of } from "rxjs/observable/of";
 
 class Counter extends React.Component {
-  static async getInitialProps ({ store, isServer }) {
-    const nextCharacterId = store.getState().nextCharacterId
-    const resultAction = await api.fetchCharacter(nextCharacterId, isServer).toPromise() // we need to convert observable to Promise
-    store.dispatch(resultAction)
+  static async getInitialProps({ store, isServer }) {
+    const resultAction = await rootEpic(
+      of(actions.fetchCharacter(isServer)),
+      store
+    ).toPromise(); // we need to convert Observable to Promise
```

Comment:
I'd probably put a little explainer above here that repeats some of what's in the README about invoking our epic to get our initial state server side.